.circleci/config.yml

```diff
@@ -133,18 +156,16 @@ jobs:
     steps:
       - checkout
 
-      - run: *apt-install
-      - run: *fonts-install
-      - run: *pip-install
+      - apt-install
+      - fonts-install
+      - pip-install
 
-      - run: *deps-install
-      - run: *mpl-install
+      - deps-install
+      - mpl-install
 
-      - run: *doc-build
+      - doc-build
 
-      - run: *doc-bundle
-      - store_artifacts:
-          path: doc/build/sphinx-gallery-files.tar.gz
```

Comment:
Do we need this artifact?

Reply:
I'm not sure; it was originally there to confirm that sphinx-gallery was working, but that seems rather rare these days. On the other hand, it really doesn't spend much time doing this at all.